internal/cypress/v1/config.go

```diff
@@ -206,6 +206,10 @@ func (p *Project) Validate() error {
 		return errors.New(msg.MissingRegion)
 	}
 
+	if ok := config.ValidateVisibility(p.Sauce.Visibility); !ok {
+		log.Warn().Msgf(msg.InvalidVisibilityWarning, p.Sauce.Visibility)
```

Comment:
Should we maybe actually fail the job? I quickly tested webdriver and I know that it doesn't fail there.

But, I'm wondering this: Imagine that someone typos the visibility setting and intentionally wants it something more restrictive. We'd default to `team`. Now, `team` is not the "worst" possible scenario here. After all, it's still only visible to people within the organization. But I do wonder, if I as a user, would rather have the application fail in this case than potentially overexpose the job?

Reply:
Yah, I think you're right. Lemme fix it up.